DesignSystem/src/styles/main.scss

```diff
@@ -1,10 +1,14 @@
 @import './reset';
 @import './variables';
 
+// Animations
+@import './Animations/animations.scss';

```

Comment:
The folders should be in lowercase. The files should be capital case

Reply:
Fixed the name of the file, but all our folders are in Capital Case, we agreed on that before when we started.
![Captura](https://user-images.githubusercontent.com/54256796/87073488-1c044400-c1e3-11ea-9bf1-c697f52a412a.PNG)

Reply:
For that reason you should use lower case in folders.